clik/external/hal_cpu/source/CMakeLists.txt

```diff
@@ -34,11 +34,6 @@ target_include_directories(hal_cpu PRIVATE
 
 target_link_libraries(hal_cpu hal_common dl pthread)
 
-hal_add_baked_data(hal_cpu
-    hal_cpu_linker_script
-    linker_script.h
-    ${HAL_CPU_SOURCE_DIR}/include/device/program.lds)
```

Comment:
This makes it so that `program.lds` is unused, right? Should it be removed?

Reply:
Oh, I hadn't seen that, `git grep program.lds clik` doesn't find any other references in there. Is it the things in `doc/overview/tutorials/` that you're referring to?

Reply:
you are right, I think it may have used it in the past, but not now